### PR TITLE
Jphenow/process group regions

### PR DESCRIPTION
**Pull request by jphenow**
* [appsv2] Show regions based on machine placement

## appv2

```
-> % flocal regions list --app appsv2
Regions [web]: den
Regions [worker]: ord
```

## Origin

Inability to see per-process-group regions made this harder: https://community.fly.io/t/different-numbers-shown-on-the-logs-compared-to-the-instance/11923/11

### CODE REVIEW

File: api/types.go [hunk 1]
```diff
@@ -340,6 +340,9 @@ type App struct {
 	ImageDetails                ImageVersion
 	LatestImageDetails          ImageVersion
 
+	// TODO: seems wrong
+	Machines struct{ Nodes []GqlMachine }
```
Comment (jphenow) [inline api/types.go:344]: Less plugged into this work recently - should I just make a type scoped to this module?

**Comment by jphenow** [comment]
@dangra it sounded like this might relate to some region things you were talking to Kurt about - let me know if this is way off the mark?

**Comment by jphenow** [comment]
note to self, need to filter the machines we're querying for by state probably.

**Comment by jphenow** [comment]
Update: regions in appsv1 are kind of a mess in general between a simple app, an app with multi-process, and then if you put volumes in the mix its a whole thing. I backed out of the appsv1 portion of this and this just adds a regions list per-group for appsv2.

**Review by dangra — APPROVED** [review]
Looks good. This command should be ported to `internal/command/regions/` but that can be done in another PR.

**Pull request merged**